src/schemas/json/function.json

```
@@ -514,6 +516,63 @@
           "description": "The text of the email."
         }
       }
+    },
+    "sqlBinding": {
+      "properties": {
+        "connectionStringSetting": {
+          "type": "string",
+          "description": "The name of the app setting that contains the SQL connection string used to connect to the database."
```

```suggestion
          "description": "A name of the app setting that contains the SQL connection string used to connect to the database\n"
```

Optional fix: `a` may sound better.
Optional fix: add link to the documentation after a `\n`.